src/test/java/Homework19.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+
+    @Test
+    @Parameters ({"BaseURL", "HomeURL"})
+    public static void deletePlaylist () throws InterruptedException {
+        launchBrowser("https://bbb.testpro.io/");
+        login("prytulan@gmail.com", "28268286nN*");
+        clickSubmit();
+        getHome("https://bbb.testpro.io/#!/home");
+        WebElement myPlaylist = driver.findElement(By.xpath("//*[@href='#!/playlist/35873']"));
```

Comment:
This locator is bad.
Try to specify locator of an exact element you want to find (example: you can find it by inner text)
Little hint (XPath) - //li[@Class="playlist playlist"]/a[text() = "putYourPlaylistNameHere"] - (There is even shorter locator you can write)